<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-MAO4QE7A" failures="4" tests="4" name="Test" time="8.315" errors="0" timestamp="2024-03-10T17:13:20 IST">
  <testcase classname="Day8.DeleteUser" name="test_deleteUser" time="0.048">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;org.testng.ITestContext.getAttribute(String)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.testng.ITestContext.getAttribute(String)" is null
at Day8.DeleteUser.test_deleteUser(DeleteUser.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test_deleteUser -->
  <testcase classname="Day8.GetUser" name="test_getuser" time="0.003">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;org.testng.ITestContext.getAttribute(String)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.testng.ITestContext.getAttribute(String)" is null
at Day8.GetUser.test_getuser(GetUser.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test_getuser -->
  <testcase classname="Day8.UpdateUser" name="test_updateUser" time="0.649">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;org.testng.ITestContext.getAttribute(String)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.testng.ITestContext.getAttribute(String)" is null
at Day8.UpdateUser.test_updateUser(UpdateUser.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test_updateUser -->
  <testcase classname="Day8.CreateUser" name="test_createUser" time="7.495">
    <failure type="java.lang.NumberFormatException" message="For input string: &amp;quot;[null, null]&amp;quot;">
      <![CDATA[java.lang.NumberFormatException: For input string: "[null, null]"
at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
at java.base/java.lang.Integer.parseInt(Integer.java:661)
at java.base/java.lang.Integer.parseInt(Integer.java:777)
at java_lang_Integer$parseInt.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
at io.restassured.internal.common.path.ObjectConverter.convertObjectTo(ObjectConverter.groovy:29)
at io.restassured.path.json.JsonPath.getInt(JsonPath.java:245)
at Day8.CreateUser.test_createUser(CreateUser.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test_createUser -->
</testsuite> <!-- Test -->
